tests/core/pyspec/eth2spec/test/electra/epoch_processing/pending_deposits/test_apply_pending_deposit.py

```diff
@@ -518,3 +536,63 @@ def test_apply_pending_deposit_success_top_up_to_withdrawn_validator(spec, state
     current_epoch = spec.get_current_epoch(state)
 
     assert spec.is_fully_withdrawable_validator(validator, balance, current_epoch)
+
+
+@with_electra_and_later
+@spec_state_test
+def test_duplicate_deposit_min_act_balance(spec, state):
+    # fresh deposit = next validator index = validator appended to registry
+    validator_index = len(state.validators)
+    amount = spec.MIN_ACTIVATION_BALANCE
+    pre_validator_count = len(state.validators)
+
+    pending_deposit1 = prepare_pending_deposit(spec, validator_index, amount, signed=True)
+    yield from run_pending_deposit_applying(spec, state, pending_deposit1, validator_index)
+    pending_deposit2 = prepare_pending_deposit(spec, validator_index, amount, signed=True)
+    yield from run_pending_deposit_applying(spec, state, pending_deposit2, validator_index)
```

Comment:
Unfortunately, we cannot yield twice in a single test case. This wouldn't work with our test generator setup.